llvm/lib/Bitcode/Writer/BitcodeWriter.cpp

```diff
@@ -1037,6 +1037,8 @@ void ModuleBitcodeWriter::writeTypeTable() {
     }
     case Type::DXILPointerTyID:
       llvm_unreachable("DXIL pointers cannot be added to IR modules");
+    case Type::SizedCapabilityTyID:
+      llvm_unreachable("Fixed-size capabilities cannot be added to IR modules");
```

Comment:
EVT::dump will print it, so IMO it's best for it to match the equivalent MVT, otherwise you get `<4 x codegen-cap-128>` and `c128` coming out of the same method depending on whether it's in a simple type.